### PR TITLE
Update to kuzzle sdk-javascript 1.7.3

### PR DESCRIPTION
**Pull request by jenow**
Tournament demo now working with kuzzle beta.4

### CODE REVIEW

File: tournament/server/tournament_server.js [hunk 1]
```diff
@@ -103,9 +103,10 @@ function createRoom () {
   subscribedRoom = kuzzleRoom.subscribe({}, {scope: 'none', users: 'none'}, function (error, data) {
     var alivePlayer = [];
 
-    switch (data._source.event) {
+    console.log('# sub[', room, ']', data);
```
Comment (ballinette) [inline tournament/server/tournament_server.js]: do we keep this console.log() ?

**Comment by ballinette** [comment]
+1

**Comment by benoitvidis** [comment]
+1

**Comment by scottinet** [comment]
+1

**Pull request merged**